src/index.ts

```diff
@@ -2,7 +2,12 @@
 let submittedForm: HTMLFormElement | null = null
 
 function shouldResumeField(field: HTMLInputElement | HTMLTextAreaElement): boolean {
-  return !!field.id && field.value !== field.defaultValue && field.form !== submittedForm
+  return (
+    !!field.id &&
```

Comment:
Small nitpick but I think the `!!` is not strictly necessary. (Feel free to keep it though, doesn't hurt either 😄 )

Reply:
For this PR I'm going to leave it because it was already there :)